### PR TITLE
Add compiler flag to see code output size

### PR DESCRIPTION
**Pull request by JonathanStarup**
--Xbenchmark-code-size

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/Main.scala [hunk 1]
```diff
@@ -155,6 +155,12 @@ object Main {
         System.exit(1)
     }
 
+    // check if the -Xbenchmark-code-size fals was passed.
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/Main.scala:158]: typo

File: main/src/ca/uwaterloo/flix/runtime/CompilationResult.scala [hunk 2]
```diff
@@ -26,7 +26,7 @@ import ca.uwaterloo.flix.language.ast._
   * @param root the abstract syntax tree of the program.
   * @param defs the definitions in the program.
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/runtime/CompilationResult.scala:27]: update doc

File: main/src/ca/uwaterloo/flix/Main.scala (hunk 1 above)
Comment (JonathanStarup) [reply main/src/ca/uwaterloo/flix/Main.scala:158]: oops. Why are they listed as `-X...` here when they are actually `--X...`?

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/Main.scala]: Typo too?

**Pull request merged**